build.gradle.kts

```diff
@@ -33,10 +33,10 @@ java {
 }
 
 dependencies {
-    compile("org.jetbrains.kotlin:kotlin-stdlib")
-    compile("org.jetbrains.kotlinx:kotlinx-coroutines-core:0.22.5")
-    compile("com.squareup.retrofit2:retrofit:2.4.0")
-    testCompile("junit:junit:4.12")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib")
```

Comment:
Please, do not use `implementation` configuration for stdlib and other dependencies. We do not want to hide stdlib from transitive dependencies 
Use compile instead (`api` configuration is not available without java-library plugin and not a better than compile in this case)